### ISSUE TITLE
Graphs tab on TensorBoard failing with a JavaScript error

### ISSUE THREAD

**Issue by mebersole** [tensorflow/tensorflow#1046]
Built from TOT today (Feb 10th) with GPU support.  TensorBoard loads and the Events tab works, but when I go to the Graph tab, I see `Uncaught TypeError: Polymer.dom(...).unobserveNodes is not a function`.  

I'm a TensorFlow rookie, so good chance I did something wrong.

**Comment by dsmilkov** [comment]
@mebersole Can you clarify "built from TOT"? Did you built it using `bazel` or using `gulp`? The right way is to run:
```
bazel build tensorflow/tensorboard
bazel-bin/tensorflow/tensorboard/tensorboard --logdir path/to/log/dir
```

I did this today and tensorboard is working.  In case you are building your own front-end (typescript, html), in which case you are running `bower install`, make sure you choose polymer 1.2.4 if bower asks you for which version of polymer. I tried running bower install from scratch, and purposely chose polymer 1.1.5 which resulted in the same error (Uncaught TypeError: Polymer.dom(...).unobserveNodes is not a function) that you have.

Hope this helps!

**Comment by mebersole** [comment]
@dsmilkov Thanks for the help.  I just cloned 967cd485 to avoid issue #1076 (I worked around it by building and installing a pip package) and built with `bazel build -c opt --config=cuda tensorflow/tensorboard`.  I then ran with `bazel-bin/tensorflow/tensorboard/tensorboard --logdir /home/ubuntu` and the Graph tab is still giving me the `Uncaught TypeError: Polymer.dom(...).unobserveNodes is not a function` error and showing a blank page other than the header bar (title and tab links).  Any suggestions?

**Comment by mebersole** [comment]
I should say I worked around #1076 when I first reported this issue.  I can now run TensorBoard directly with bazel.

**Comment by mebersole** [comment]
Also, I have this set up on a test AWS instance.  If it's easier, I can pass you the login information and you can look at it live.

**Comment by mikehaley** [comment]
For what it's worth I'm having exactly the same problem with the latest build.    Building on Ubuntu 14.04.2 and running TensorBoard using bazel.

**Comment by mebersole** [comment]
@mikehaley did you build with GPU support `-c opt --config=cuda`?  Maybe that will help narrow down the issue.

**Comment by mikehaley** [comment]
Yes.  Built with CUDA.   Also updated polymer to 1.2.4 @dsmilkov suggested.

**Comment by dsmilkov** [comment]
Hi guys,

If you open the console while your browser is pointed to 0.0.0.0:6006 and type Polymer.version do you get 1.1.5 or 1.2.4 or something else?

If you get 1.1.5 when you need to update to 1.2.4. To do this, download polymer 1.2.4. Unzip and copy the 3 html files (polymer*.html) into the external polymer directory under site-packages dir: /usr/lib/python2.7/site-packages/external/polymer (overwrite the previous files).

Let me know how it goes. We will also be releasing a patch for 0.7.0 where we ship Polymer 1.2.4 with it.

**Comment by mebersole** [comment]
Going to 1.2.4 of polymer worked for me.

**Comment by mikehaley** [comment]
Thanks @dsmilkov.   Forcing polymer to 1.2.4 works.   It was indeed running 1.1.5 before despite the settings in bower.json.

**Comment by dsmilkov** [comment]
Closed in #1134
